components/Map.js

```diff
@@ -59,7 +77,80 @@ class Map extends React.Component {
       },
     });
 
+    // In order to add a geocoder to our map, we need a mapbox access token.
+    // We've stored that using environment variables inside out nextjs config file.
+    const { publicRuntimeConfig } = getConfig();
+    const accessToken = publicRuntimeConfig.MapboxAccessToken;
+
+    const geocoder = new MapboxGeocoder({
+      accessToken: accessToken,
+      flyTo: true,
+      placeholder: 'Show projects near...',
```

Comment:
Ellipsis character: …

Reply:
Didn't know that was a thing! Updated it.